frontend/packages/console-app/src/components/nodes/NodeLogsUnitFilter.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { Chip, ChipGroup, TextInput } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { getQueryArgument } from '@console/internal/components/utils';
+
+type NodeLogsUnitFilterProps = {
+  onChangeUnit: (value: string) => void;
+  unit: string;
+};
+
+const NodeLogsUnitFilter: React.FC<NodeLogsUnitFilterProps> = ({ onChangeUnit }) => {
+  const firstRender = React.useRef(true);
+  const inputRef = React.useRef<HTMLInputElement>();
+  const [values, setValues] = React.useState<string[]>(getQueryArgument('unit')?.split(',') || []);
+  const { t } = useTranslation();
+
+  React.useEffect(() => {
+    const input = inputRef.current;
+    const listener = (event) => {
+      const { value } = event.currentTarget;
+      if ((event.code === 'Enter' || event.code === 'NumpadEnter') && value !== '') {
+        event.preventDefault();
+        setValues((prevValues) => [...prevValues, value]);
```

Comment:
Missed this in my PR, but we should probably call `_.uniq` to make sure we don't have the same unit twice.

```suggestion
        setValues((prevValues) => _.uniq([...prevValues, value]));
```